backend/lpconversion/models.py

```diff
@@ -41,6 +42,43 @@
 }
 
 
+class LpSellOrder(models.Model):
+    """
+    A request to sell loyalty points.
+    """
+
+    status_choices = (
+        ("pending", "Pending"),
+        ("accepted", "Accepted"),
+        ("sent", "LP Sent"),
+        ("paid", "Paid"),
+        ("closed", "Closed"),
+    )
+    status = models.CharField(
+        max_length=10, choices=status_choices, default="pending"
+    )
+    seller = models.ForeignKey(User, on_delete=models.CASCADE)
+    loyalty_points = models.IntegerField(default=0)
+    rate = models.IntegerField(default=0)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+    discord_thread_id = models.BigIntegerField(blank=True, null=True)
+
+
+class LpBuyOffer(models.Model):
```

Comment:
What are the thoughts behind this versus attaching the buyer to the sell order? We seem to have multiple rates this way which isn't really how the conversion team works today.